xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -502,6 +502,47 @@ PVR_ERROR CPVRClients::RenameTimer(const CPVRTimerInfoTag &timer, const std::str
   return error;
 }
 
+PVR_ERROR CPVRClients::GetTimerTypes(CPVRTimerTypes& results) const
+{
+  PVR_ERROR error(PVR_ERROR_NO_ERROR);
+
+  PVR_CLIENTMAP clients;
+  GetConnectedClients(clients);
+
+  for (const auto &clientEntry : clients)
```

Comment:
cosmetics, but you're putting the deref op next to the type in the function header, and next to the name here and in some other bits. personally I prefer it next to the type, as that's what it's modifying, not the name.